proctor-codegen-test/pom.xml

```diff
@@ -18,7 +18,7 @@
     <build>
         <plugins>
             <plugin>
-                <groupId>${project.groupId}</groupId>
```

Comment:
There are some other pom files that still have references to "${project.groupId}".

Should we be replacing all of these with com.indeed ?